src/main/java/com/selfxdsd/todocli/Todo.java

```diff
@@ -31,9 +31,6 @@
  * @checkstyle AbbreviationAsWordInName (500 lines)
  * @checkstyle FinalParameters (500 lines)
  * @since 0.0.1
- * @todo #4:60min  Add interface Todos which should represent
```

Comment:
@amihaiemil This puzzle is actually solved by this issue, odd that wasn't picked by pdd.

And looking over `Todo`, maybe for the future we should making it immutable and leave mutability for a `Todo` builder; it feels that are too many constructors (3) :D

Reply:
@criske 

> And looking over Todo, maybe for the future we should making it immutable and leave mutability for a Todo builder (in TodoParser); it feels that are too many constructors (3) :D

Yes, definitely. 

At the moment 0pdd doesn't pick up the puzzles from this repo because it complains about puzzle errors. The word "todo" appears in a lot of places in this project and pdd complains about it... 

We will have to switch to this utilitary soon and then it will work.